srcpkgs/uriparser/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'uriparser'
+pkgname=uriparser
+version=0.8.6
+revision=1
+wrksrc=${pkgname}-${version}
+build_style=gnu-configure
+configure_args="--disable-test"
+hostmakedepends="pkg-config"
+makedepends="doxygen graphviz qt5-tools cpptest-devel"
```

Comment:
`doxygen graphviz qt5-tools` should be in `hostmakedepends`

Reply:
Fixed.